.github/workflows/ci.yml

```
@@ -9,8 +9,8 @@ jobs:
     strategy:
       max-parallel: 2
       matrix:
-        os: [ubuntu-latest, macOS-latest]
-        python-version: [3.9, 3.10]
+        os: [ubuntu-latest, macos-latest]
+        python-version: ["3.11", "3.12"]
```

```suggestion
        python-version: ["3.11", "3.12"]
        exclude:
          - python-version: '3.11'
            os: macos-latest
```
Maybe only 3.12 works.  Not sure why (maybe would need to not use `-latest`?)